src/main/resources/org/cbioportal/persistence/mybatis/ClinicalEventMapper.xml

```diff
@@ -181,7 +181,7 @@
                 </if>
             </if>
         </where>
-        Group by clinical_event.EVENT_TYPE, patient.STABLE_ID
+        Group by clinical_event.EVENT_TYPE, patient.INTERNAL_ID
```

Comment:
@haynescd this is one fix i did.  i think we can keep it.   the stable id is not unique across study

Reply:
agreed